tests/release/pipelines/release_to_github.go

```diff
@@ -0,0 +1,199 @@
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = fw.AsKubeAdmin.ReleaseController.CreateReleasePipelineRoleBindingForServiceAccount(managedNamespace, managedServiceAccount)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = fw.AsKubeAdmin.CommonController.CreateRegistryAuthSecret(releasecommon.RedhatAppstudioUserSecret, managedNamespace, sourceAuthJson)
+		Expect(err).ToNot(HaveOccurred())
+
+		err = fw.AsKubeAdmin.CommonController.LinkSecretToServiceAccount(devNamespace, releasecommon.HacbsReleaseTestsTokenSecret, constants.DefaultPipelineServiceAccount, true)
+		Expect(err).ToNot(HaveOccurred())
+
+		err = fw.AsKubeAdmin.CommonController.LinkSecretToServiceAccount(managedNamespace, releasecommon.RedhatAppstudioUserSecret, constants.DefaultPipelineServiceAccount, true)
+		Expect(err).ToNot(HaveOccurred())
+
+		publicKey, err := fw.AsKubeAdmin.TektonController.GetTektonChainsPublicKey()
+		Expect(err).ToNot(HaveOccurred())
+		Expect(fw.AsKubeAdmin.TektonController.CreateOrUpdateSigningSecret(
+			publicKey, releasecommon.PublicSecretNameAuth, managedNamespace)).To(Succeed())
+
+		defaultECP, err := fw.AsKubeAdmin.TektonController.GetEnterpriseContractPolicy("default", "enterprise-contract-service")
+		Expect(err).NotTo(HaveOccurred())
+		policy := contract.PolicySpecWithSourceConfig(defaultECP.Spec, ecp.SourceConfig{Include: []string{"@minimal"}, Exclude: []string{"cve"}})
+
+		_, err = fw.AsKubeAdmin.HasController.CreateApplication(releasecommon.ApplicationNameDefault, devNamespace)
+		Expect(err).NotTo(HaveOccurred())
+
+		GitSourceComponentUrl := "https://github.com/jinqi7/terraform-provider-rhcs/"
+
+		componentObj := appstudioApi.ComponentSpec{
+			ComponentName: releasecommon.ComponentName,
+			Application:   releasecommon.ApplicationNameDefault,
+			Source: appstudioApi.ComponentSource{
+				ComponentSourceUnion: appstudioApi.ComponentSourceUnion{
+					GitSource: &appstudioApi.GitSource{
+						URL:      GitSourceComponentUrl,
+						Revision: "release2.1",
+					},
+				},
+			},
+		}
+		// Create a component with Git Source URL, a specified git branch
+		component, err = fw.AsKubeAdmin.HasController.CreateComponent(componentObj, devNamespace, "", "", releasecommon.ApplicationNameDefault, false, utils.MergeMaps(constants.ComponentPaCRequestAnnotation, constants.ImageControllerAnnotationRequestPublicRepo))
+		Expect(err).ShouldNot(HaveOccurred())
+
+//		component = releasecommon.CreateComponentByCDQ(*fw, devNamespace, managedNamespace, releasecommon.ApplicationNameDefault, releasecommon.ComponentName, GitSourceComponentUrl)
+		_, err = fw.AsKubeAdmin.ReleaseController.CreateReleasePlan(releasecommon.SourceReleasePlanName, devNamespace, releasecommon.ApplicationNameDefault, managedNamespace, "")
+		Expect(err).NotTo(HaveOccurred())
+
+		data, err := json.Marshal(map[string]interface{}{
+			"mapping": map[string]interface{}{
```

Comment:
This test relies on other things. See https://github.com/redhat-appstudio/release-service-catalog/blob/development/tasks/collect-gh-params/tests/test-collect-gh-params.yaml
I am not sure the mapping is used in this pipeline

Reply:
Thanks for the information.